### DISCUSSION TITLE
[EXAMPLE] Summarize your office hours topic here (Office hours: Sept 22, 2020)

### DISCUSSION THREAD

**Discussion by serifluous** [twilio-labs/paste#770] [Office hours] [answered]
**The specific questions I have are...**

- How can I style the options in Combobox?
- How do I use the new unified theme in Sketch?
- (See other questions in the sign-up sheet)

**Background:** Following up on Discussion #677. We are planning to add a Widget search feature in Studio and need to show multiple text styles in the results. We're also planning for the UDL theme switch.
**Stage of design:** Exploration
**Research:** (in sign-up sheet)
**Links to mockups or prototype:** https://drive.google.com/file/d/1jOYiOH19bq-fOQQi08oHa3ybcARGBSE8/view?usp=sharing and https://codesandbox.io/s/paste-starter-kit-rj7yy

-----------
Markup to copy:

```
**The specific questions I have are...**

- Question
- Question

**Background:** Following up on Discussion #677. We are planning to add a Widget search feature in Studio and need to show multiple text styles in the results. We're also planning for the UDL theme switch.
**Stage of design:** Exploration
**Research:** (in sign-up sheet)
**Links to mockups or prototype:** https://www.google.com/
```

**Comment by loreina** [comment, marked as answer]
Marking as answered
